src/styles.css

```diff
@@ -248,3 +248,19 @@ main {
     display: block;
   }
 }
+
+.leaderboard {
+  text-align: center;
+}
```

Comment:
I think we should add some space above the leaderboard, this could maybe be done by adding this class to the leaderboard's container and adding something like margin-top: 50px